reports-scheduler/build.gradle

```diff
@@ -57,7 +57,7 @@ apply plugin: 'idea'
 apply plugin: 'opensearch.opensearchplugin'
 apply plugin: 'opensearch.pluginzip'
 apply plugin: 'opensearch.testclusters'
-apply plugin: 'io.gitlab.arturbosch.detekt'
+// apply plugin: 'io.gitlab.arturbosch.detekt'
```

Comment:
could you remove the comments? it's hard to tell from code why they are commented. you can create an issue to track snakeyaml and revert #517 when it's fixed, or add this PR url in the TODO comment

Reply:
Removed all comments and will track enabling detekt with https://github.com/opensearch-project/dashboards-reports/issues/520